rpcs3/Emu/Cell/PPUInterpreter.cpp

```diff
@@ -1530,7 +1501,7 @@ bool ppu_interpreter_precise::VPKSWSS(ppu_thread& ppu, ppu_opcode_t op)
 {
 	const auto& a = ppu.vr[op.va];
 	const auto& b = ppu.vr[op.vb];
-	auto& d = ppu.vr[op.vd];
+	v128 d;
```

Comment:
Good catch but there are also some other instruction with similar problem missing.

Reply:
Hopefully I've covered all the problematic cases.
In cases such as https://github.com/RPCS3/rpcs3/blob/master/rpcs3/Emu/Cell/PPUInterpreter.cpp#L365
it doesnt matter because of the way the register is being treated in equal sized slices between source and destination registers.